modules/migration-understanding-cpma.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+// migration/migrating-openshift-3-to-4.adoc
+[id='migration-understanding-cpma_{context}']
+= Understanding the Control Plane Migration Assistant
+
+The Control Plane Migration Assistant (CPMA) is a CLI-based tool that assists you in migrating the control plane from {product-title} 3.x to 4.2. The CPMA processes {product-title} 3.x configuration files and generates Custom Resource (CR) manifest files, which are consumed by {product-title} 4.2 Operators.
+
+Because {product-title} 3 and 4 have significant configuration differences, not all parameters are processed. CPMA generates a report that describes whether features are supported fully, partially, or not at all.
+
+.Configuration files
+
+CPMA uses the Kubernetes and {product-title} APIs to access the following configuration files on an {product-title} 3 cluster:
+
+* Master configuration file (default: `/etc/origin/master/master-config.yaml`)
+* CRI-O configuration file (default: `/etc/crio/crio.conf`)
+* etcd configuration file (default: `/etc/etcd/etcd.conf`)
+* Image registries file (default: `/etc/containers/registries.conf`)
+* Dependent configuration files:
+** Password files (for example, HTPasswd)
+** ConfigMaps
+** Secrets
+
+.CR Manifests
+
+CPMA generates CR manifests for the following configurations:
+
+* API server CA certificate: `100_CPMA-cluster-config-APISecret.yaml`
++
+[NOTE]
+====
+If you are using an unsigned certificate, you must link:../authentication/certificates/api-server.html#add-named-api-server_api-server-certificates[add the API server certificate manually] to the target cluster.
```

Comment:
xref:../authentication/certificates/api-server.adoc#add-named-api-server_api-server-certificates[add the API server certificate manually]